### PR TITLE
titles are now capitalized og images

### PR DESCRIPTION
**Pull request by Isaiah-Hamilton**
## What kind of change does this PR introduce?

titles are now capitalized in og images

## What is the current behavior?

titles not capitalized

## What is the new behavior?

titles are capitalized

### CODE REVIEW

File: supabase/functions/og-images/component/Docs.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ const Docs = (props: Props) => {
         </div>
         <div tw="flex flex-col">
           <div tw="flex flex-col absolute left-[64px] top-[154px] w-[1072px]">
-            <h1 tw="my-0 mb-2 text-[60px]">{title}</h1>
+            <h1 tw="my-0 mb-2 text-[60px]">{title.substring(0, 1).toUpperCase() + title.substring(1)}</h1>
```
Comment (MildTomato) [inline supabase/functions/og-images/component/Docs.tsx:45]: just note, if title is not defined, this might fail

**Review by MildTomato — APPROVED** [review]

File: supabase/functions/og-images/component/Docs.tsx (hunk 1 above)
Comment (kiwicopple) [inline supabase/functions/og-images/component/Docs.tsx:45]:
```suggestion
            <h1 tw="my-0 mb-2 text-[60px]">{!title ? 'Supabase' : title.substring(0, 1).toUpperCase() + title.substring(1)}</h1>
```

Comment (kiwicopple) [reply supabase/functions/og-images/component/Docs.tsx:45]: what about now @MildTomato

Comment (MildTomato) [reply supabase/functions/og-images/component/Docs.tsx]: 🔥

**Pull request merged**